### ISSUE TITLE
ContextMenu should narrarate out index and item count as you navigate through

### ISSUE THREAD

**Issue by rickyp-ms** [OfficeDev/office-ui-fabric-react#2432]
### Bug Report

- __Package version(s)__: 4.21.3
- __Browser and OS versions__: Edge, IE

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: Normal

Products/sites affected: tasks.office..om

#### Describe the issue:

When using Narrator arrowing through the context menu should announce how many elements there are to arrow through along with what index you are currently on.

#### Actual behavior:

You're told it's a menu item but nothing else

#### Expected behavior:

See description of issue
